.env

```diff
@@ -0,0 +1,2 @@
+USERNAME=arsenhovhannisyan
```

Comment:
.env file should not be committed to the repo with real data.
You may leave keys empty. In that case, you will indicate which keys should be set, but will not share your real sensitive data.